0.data_analysis/nbconverted/plates_1_2_sig_test_nf1_sc_all_cellprofiler.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Obnibus and post hoc testing
+
+# ## Imports
+
+# In[1]:
+
+
+import sys
+from sklearn.ensemble import IsolationForest
+import scikit_posthocs as sp
+import pathlib
+import numpy as np
+import pandas as pd
+
+rel_root = pathlib.Path("..")
+sys.path.append(f'{rel_root}/utils')
+import analysis_utils as au
+import preprocess_utils as ppu
+
+
+# # Preprocess data using preprocess utils
+
+# In[2]:
+
+
+filename = 'nf1_sc_all_cellprofiler.csv.gz'
+po1 = ppu.preprocess_data(1, filename, rel_root, ['Metadata_genotype'])
+po2 = ppu.preprocess_data(2, filename, rel_root, ['Metadata_genotype'])
+
+
+# In[3]:
+
+
+rnd_val = po1.rnd_val # Random value for all seeds
```

Comment:
if this is a constant consider using all caps and declaring at top of document under paths
```suggestion
RND_VAL = po1.rnd_val # Random value for all seeds
```

Reply:
Do you think it would be better to just set this variable to 0 each time? Otherwise, it has to come before the the object initialization of `po1`. I was also considering adding it to an env file, but that may excessive.

Reply:
I think this is probably fine